### PR TITLE
Minor copy, formatting changes for patient link DOB verification page

### LINKED ISSUE #3332: Add facility name, expiration date to patient link DOB verification screen
### WHY?

**AS A**
Patient or individual receiving test results on behalf of another,

**I WANT**
To accurately identify the test for which the results are being viewed, and to have information about expiration constraints

**SO THAT**
I can be certain that I am viewing the correct test results and am aware of the expiration date

---

**GIVEN**
The patient experience date of birth verification screen

**WHEN**
the patient navigates to the screen

**THEN** 
they will see the name of the facility that conducted the test as well as the date that the link will expire

**Text notification, verification screen, and email [copy here](https://docs.google.com/document/d/1XBcBnS-kjXVbT44q7RN_6Nx9QmdFr-aNFj_IjaV3UhE/edit#).**

### Translations
- [ ] ~Translations are required~
- [ ] ~Translation request has been submitted~
- [ ] ~Translations have been received~

### PR DESCRIPTION
**Pull request by nathancrtr**
## Related Issue

- Closes #3332 

## Changes Proposed
- @Rebecca-LM requested some further changes after #3332 was shipped
- See the actual diff for the particulars

## Testing
- I will deploy to `dev` or `test` for reviewers to interact with the changes
 
## Screenshots / Demos
- Screen Shot 2022-03-29 at 3 29 19 PM

### Design
- [ ] Any UI/UX changes have a designer as a reviewer, and changes have been approved
- [ ] Any large-scale changes have been deployed to `test`, `dev`, or `pentest` and smoke-tested by both the engineering and design teams

### Content
- [ ] Any content changes have been approved by content team

### CODE REVIEW

File: frontend/src/lang/es.ts [hunk 1]
```diff
@@ -484,7 +484,7 @@ export const es: LanguageConfig = {
         },
       },
       dob: {
-        header: "Verificar fecha de nacimiento",
+        header: "Access your COVID-19 test result (translate me!)",
```
Comment (nathancrtr) [inline frontend/src/lang/es.ts:487]: @Rebecca-LM let's don't forget about this 😂

Comment (Rebecca-LM) [reply frontend/src/lang/es.ts:487]: Acceda al resultado de su prueba de COVID-19

**Review by Rebecca-LM — DISMISSED** [review]
LGTM. @nathancrtr were you able to also update the TOS that users see when logging in to view their result?

**Comment by nathancrtr** [comment]
> LGTM. @nathancrtr were you able to also update the TOS that users see when logging in to view their result?

@Rebecca-LM Are there any changes that you're definitely expecting to see? I'm using a program to compare the [canonical TOS](https://docs.google.com/document/d/19PFqZZpeznFQ3-_TCvvF9pbMGj35m8_C/edit#) from the Drive with the version that users see and I'm not seeing any differences in the content 🕵️‍♂️

**Comment by Rebecca-LM** [comment]
> > LGTM. @nathancrtr were you able to also update the TOS that users see when logging in to view their result?
> 
> @Rebecca-LM Are there any changes that you're definitely expecting to see? I'm using a program to compare the [canonical TOS](https://docs.google.com/document/d/19PFqZZpeznFQ3-_TCvvF9pbMGj35m8_C/edit#) from the Drive with the version that users see and I'm not seeing any differences in the content 🕵️‍♂️

@nathancrtr the only thing I spotted was the date at the top shown to users was 12/4/2020. I didn't read beyond that b/c I assumed the copy was from that date, and not our current copy. If you're not seeing any differences, I'd think that just the date needs to be updated/fixed to 10/18/21.

**Review by vz3 — APPROVED** [review]

**Review by Rebecca-LM — APPROVED** [review]

**Pull request merged**